src/rezplugins/shell/_utils/powershell_base.py

```diff
@@ -202,6 +202,11 @@ def _record_shell(ex, files, bind_rez=True, print_msg=False):
         # Suppresses copyright message of PowerShell and pwsh
         cmd += ["-NoLogo"]
 
+        # Powershell execution policy overrides
+        execution_policy = self.settings.execution_policy
+        if execution_policy:
```

Comment:
I am tempted to say that we should sanitize this a little bit. Maybe use a simple regex like `re.match(r'[a-zA-Z]+', execution_policy)` to make sure nobody injects a `;` or whatever weird thing that could lead to other even more weird to debug errors.